app/assets/stylesheets/components/_tables.scss

```diff
@@ -0,0 +1,9 @@
+.BasicTable {
+  @include standard-table(100%, 40%, 25%);
+  max-width: 1200px;
```

Comment:
Do we need this `max-width`? If we do, can/should we express it as `max-width: $app-width;`?